src/pages/Home/Home.tsx

```diff
@@ -1,15 +1,40 @@
 import { type FC } from 'react';
 
-import { Container } from './styles';
+import Progress from '../../components/ProgressBar';
```

Comment:
I'd encourage naming the importing variable just like the exported name (ProgressBar instead of Progress)